html-css/week1/Ali Sharifi/order-desktop/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Payment Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="payment-container">
+      <div class="order-summary">
+        <h2>Order Summary</h2>
+<div class="price-details">
+          <div class="line">
+            <p class="total">Total</p>
+            <p class="price">$333 / month</p>
+          </div>
+          <hr class="divider" />
+          <div class="line">
+            <p class="total">Total</p>
+            <p class="price">$100</p>
+          </div>
```

Comment:
This might might be more semantic as this:
```suggestion
          <p class="line">
            <span class="total">Total</span>
            <span class="price">$100</span>
          </p>
```